deepchem/feat/molecule_featurizers/one_hot_featurizer.py

```diff
@@ -151,7 +156,11 @@ def pad_string(self, string: str) -> str:
     str
       String space padded to self.pad_length
     """
-    return string.ljust(self.max_length)
+    if isinstance(self.max_length, int):
+      return string.ljust(self.max_length)
+    else:
+      logger.info("DID NOT PAD: max_length is not int.")
```

Comment:
This warning could spam a lot of logging messages. There's a balance to strike in that we don't want to flood user logs with messages while also returning useful information

Reply:
Sounds good.